eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -99,12 +113,12 @@ object Graph {
                         ignoredVertices: Set[PublicKey],
                         extraEdges: Set[GraphEdge],
                         pathsToFind: Int,
-                        wr: WeightRatios,
+                        wr: Either[WeightRatios, HeuristicsConstants],
```

Comment:
I think we should introduce some kind of `sealed trait ScoringHeuristics` to allow future extensibility (and figure out a way of making `WeightRatios` and `HeuristicsConstants` fit below this new sealed trait).

Ideally in the longer term we'll figure out the right abstract methods this trait should have to avoid switching between different heuristics directly inside the dijkstra logic. Sorry if this is very hand-wavy, it's a bit foggy for now but I think it's a good general direction to go towards.